core/blockchain.go

```diff
@@ -197,7 +197,15 @@ func (self *BlockChain) loadLastState() error {
 			self.currentFastBlock = block
 		}
 	}
-	// Issue a status log and return
+	// Initialize a statedb cache to ensure singleton account bloom filter generation
```

Comment:
Leftover from a combi pr addressing multiple issues. I'd need to manually rewrite some stuff to get this init out of here. Since it doens't do any harm I just left it. Can clean if you want.